nautobot/docs/installation/install_system.md

```diff
@@ -0,0 +1,475 @@
+# Virtual Machine / Host Install
+
+The documentation assumes that you are running one of the following:
+
+- Ubuntu 20.04+
+- Debian 11+
+- RHEL/CentOS 8.2+
+    - Also includes other derivatives of RHEL such as RockyLinux or AlmaLinux
+
+## Install System Packages
+
+Install the prerequisite system libraries and utilities.
+
+This will install:
+
+- Git
+- Python 3
+- Pip
+- Redis server and client
+
+=== "Ubuntu/Debian"
+
+    ```no-highlight
+    sudo apt update -y
+    sudo apt install -y git python3 python3-pip python3-venv python3-dev redis-server
```

Comment:
Not blocking for this PR but I do notice that we have some drift between what we recommend installing here versus what we install in our Docker build - the Docker build adds:

- libxml2
- mime-support
- openssl
- libxmlsec1-dev
- libxmlsec1-openssl

and also:

- build-essential
- libssl-dev
- pkg-config
- libldap-dev
- libsasl2-dev

IIRC most of these are specific to optional "extras" dependencies like SSO and LDAP but we should probably document them somewhere in this file in the future.